internal/runtime/client/client.go

```diff
@@ -153,6 +160,89 @@ func (c *client) Unregister(extensionConfig *runtimev1.ExtensionConfig) error {
 	return nil
 }
 
```

Comment:
Let's please add some unit tests for Call and CallAll. If I see correctly it should be relatively straightforward (what we've done for httpCall + we have to setup a registry (by just registering stuff and skipping discovery)